sigmastate/src/main/scala/sigmastate/lang/SigmaCompiler.scala

```diff
@@ -124,6 +125,10 @@ class SigmaCompiler(settings: CompilerSettings) {
         MethodCall(xs,
           SCollection.GetOrElseMethod.withConcreteTypes(Map(tIV -> xs.tpe.elemType)),
           Vector(index, default), Map())
+      case LastBlockUtxoRootHash =>
+        // Dummy implementation to satisfy compiler, current one is commented below:
+        MethodCall(Global, SGroupElement.MultiplyMethod, Vector(), Map())
+        // MethodCall.typed[Value[SAvlTree.type]]( ValUse(1, SContext), SContext.lastBlockUtxoRootHashMethod, Vector(), Map())
```

Comment:
use this `MethodCall(Context, SContext.lastBlockUtxoRootHashMethod, Vector(), Map())`